### PR TITLE
add link for protocols in traffic management

### PR DESCRIPTION
**Pull request by visheshtha**
Added a link for protocols in traffic management in feature status page.

### CODE REVIEW

**Comment by visheshtha** [comment]
Hi @kfaseela,PTAL !

**Review by kfaseela — APPROVED** [review]

**Comment by kfaseela** [comment]
Please take a look at the failing check

**Comment by visheshtha** [comment]
Hi @kfaseela ,thanks for approving ,I am not sure why this check is failing,will take a look

**Comment by visheshtha** [comment]
The test failure is tracked under https://github.com/istio/test-infra/pull/4949

File: features.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 features:
   - name: "Protocols:HTTP1.1/HTTP2/gRPC/TCP"
     id: "traffic.http_protocols"
+    link: "https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/"
```
Comment (ilrudie) [inline features.yaml:4]: I think you only need part of the path.

```suggestion
    link: "/docs/ops/configuration/traffic-management/protocol-selection/"
```

**Review by ilrudie — COMMENTED** [review]
Looks like a good addition but I think you don't need the full URL based on the formatting of the rest of the file. Someone who knows more might correct me though.

File: features.yaml (hunk 1 above)
Comment (visheshtha) [reply features.yaml:4]: sure @ilrudie,will update

**Comment by visheshtha** [comment]
Hi @ilrudie,changed the path.

**Review by ericvn — APPROVED** [review]

**Pull request merged**